client/components/project-version-select/select.template.html.slim

```diff
@@ -9,7 +9,7 @@
 
   / dropdown menu to select an existing version
   ui-select id='{{ prefix }}-project-version-select' ng-if='!config.newVersion && modelProperty' ng-model='modelObject[modelProperty]' reset-search-input=true
-    ui-select-match placeholder='{{ getPlaceholder() }}' allow-clear='{{ !!latestVersion }}' {{ $select.selected.name }}
+    ui-select-match placeholder='{{ getPlaceholder() }}' allow-clear='{{ allowClear || !!latestVersion }}' {{ $select.selected.name }}
```

Comment:
If we now have an explicit `allowClear` option for the directive, then I think that if it's set, it alone should determine whether the clear button is shown. That would be more intuitive.

You could set a default depending on whether the latest version is given, but if I use this directive with `allow-clear='false'`, I expect no clear button to be shown.